addon/services/device/layout.js

```diff
@@ -96,24 +96,32 @@ export default Service.extend(Evented, {
     }
     let oldWidth = this.get('width');
     let oldHeight = this.get('height');
-    let width = Math.max(document.documentElement.clientWidth, window.innerWidth || 0);
-    let height = Math.max(document.documentElement.clientHeight, window.innerHeight || 0);
+    let width = this._currentWidth();
+    let height = this._currentHeight();
 
     this.setProperties({
       width,
       height
     });
 
     if (oldWidth !== width) {
-      this.trigger('width-changed')
+      this.trigger('width-change');
     }
 
     if (oldHeight !== height) {
-      this.trigger('height-changed');
+      this.trigger('height-change');
     }
 
     if ((oldWidth !== width) || (oldHeight !== height)) {
-      this.trigger('resized');
+      this.trigger('resize');
     }
+  },
+
+  _currentWidth() {
```

Comment:
I tried to just stub out `document.documentElement.clientWidth` using `td.replace` but apparently that's not feasible, so I added an abstraction layer here that allows us to stub the "current" size of the browser in the tests to ensure that the tests are reliable.